liubai-backends/liubai-laf/cloud-functions/service-send.ts

```
@@ -269,6 +274,151 @@ export class LiuTencentSMS {
+    }
+    const mac = new qiniu.auth.digest.Mac(ak, sk)
+    return mac
+  }
+
+  static sendVerifyCode(
+    code: string,
+    mobile: string,   // 11 digits
+  ): Promise<LiuRqReturn> {
+    // 1. get mac
+    const mac = this._getMac()
+    if(!mac) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)